src/secp256k1/src/java/org/bitcoin/NativeSecp256k1Test.java

```diff
@@ -52,7 +52,7 @@ public static void testSecKeyVerifyPos() throws AssertFailException{
     }
 
     /**
-      * This tests secret key verify() for a invalid secretkey
+      * This tests secret key verify() for an invalid secretkey
```

Comment:
These changes should be reported upstream. Please skip the changes made to this file.

Reply:
Fixed.